js/ui/boxpointer.js

```diff
@@ -80,6 +80,36 @@ var BoxPointer = new Lang.Class({
         }
     },
 
+    // BoxPointer.show() and BoxPointer.hide() are here for only compatibility
+    // purposes, and will be removed in 3.32.
+    show(animate, onComplete) {
```

Comment:
We don't really need this commit though, right? Since we don't care about extensions.